procedures/lvv-t1802.ipynb

```diff
@@ -0,0 +1,784 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Test Case LVV-T1802\n",
+    "This case will verify that the integration of the M2 Hexapod with SAL.\n",
+    "The blocks below represent the steps of the test case.\n",
```

Comment:
Why do we need separate notebooks for integration of camera hexapod vs M2 hexapod? I worry that this will result in far too much duplicated code and eventual divergence of the tests.

Reply:
Right, this more for historical reasons. They could come together with some generalization. Depending on how often we need these notebooks, I will do this later.